xbmc/guilib/DispResource.h

```diff
@@ -27,5 +27,11 @@ class IDispResource
   virtual void OnLostDisplay() {};
   virtual void OnResetDisplay() {};
   virtual void OnAppFocusChange(bool focus) {};
+  virtual void FrameMove() {};
 };
 
+class IRenderLoop
```

Comment:
Can you add a virtual destructor?

Reply:
objects of this type are not supposed to be deleted.

Reply:
what about RetroPlayer? players are destroyed when the file is closed

Reply:
players are not destroyed using the type IRenderLoop

Reply:
ok that makes sense. putting this in the documentation would help.

Reply:
See my comment re. removal of virtual ~IDispResource() some lines above...